core/ledger/kvledger/txmgmt/privacyenabledstate/test_exports.go

```diff
@@ -100,6 +112,22 @@ type CouchDBCommonStorageTestEnv struct {
 	couchCleanup      func()
 }
 
+// StartExternalResource starts external couchDB resources.
+func (env *CouchDBCommonStorageTestEnv) StartExternalResource() {
+	if env.couchAddress == "" {
+		env.couchAddress = env.setupCouch()
+	}
+}
+
+// StopExternalResource stops external couchDB resources.
+func (env *CouchDBCommonStorageTestEnv) StopExternalResource() {
+	csdbProvider, _ := env.provider.(*CommonStorageDBProvider)
```

Comment:
If you're not going to test that the type assertion is correct, then no need to `, _`, `myVar := foo.(*myType)` should be fine.